core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy

```diff
@@ -29,22 +29,21 @@ import hudson.model.AdministrativeMonitor
 f = namespace(lib.FormTagLib)
 st = namespace("jelly:stapler")
 
-f.section(title: _("Administrative monitors configuration")) {
+f.section(title: _("Administrative monitors"), description: _("blurb")) {
     f.advanced(title: _("Administrative monitors")) {
-        f.entry(title: _("Enabled administrative monitors")) {
-            p(class: "jenkins-form-description", _("blurb"))
+        f.entry() {
             for (AdministrativeMonitor am : new ArrayList<>(AdministrativeMonitor.all())
                     .sort({ o1, o2 -> o1.getDisplayName() <=> o2.getDisplayName() })) {
-                div(class: "jenkins-checkbox-help-wrapper") {
-                    f.checkbox(name: "administrativeMonitor",
-                            title: am.displayName,
-                            checked: am.enabled,
-                            json: am.id)
-                    if (am.isSecurity()) {
-                        span(style: 'margin-left: 0.5rem', class: 'am-badge', _("Security"))
+                div(style: "margin-bottom: 0.625rem") {
```

Comment:
any reason not to use one of the existing classes for spacing?

Reply:
They're too large - smallest is 1rem. Might be an idea to have small spacing classes.